docker-compose.yaml

```diff
@@ -50,7 +50,7 @@ services:
       - type: bind
         source: ./frontend/build
         target: /frontend/build
-    command: sh -c "yarn start-docker"
+    command: sh -c "yarn prebuild && yarn start-docker"
```

Comment:
Doesn't `yarn start-docker` redirect to `ng serve`/`ng build`? If so, `yarn preserve`/`yarn prebuild` should run implicitly.

Reply:
It does not do `yarn serve` but `ng serve`, so that's why (I think) the preserve is not triggered.